src/sendCommercialMetrics.spec.ts

```diff
@@ -19,39 +44,108 @@ describe('sendCommercialMetrics', () => {
 	});
 
 	it('send commercial metrics success', () => {
-		Object.defineProperty(document, 'visibilityState', {
-			value: 'hidden',
-			writable: true,
-		});
-		expect(
-			sendCommercialMetrics('page view id', 'browser id', true),
-		).toEqual(true);
+		setVisibility();
+
+		expect(mockSendMetrics()).toEqual(true);
 
 		expect((navigator.sendBeacon as jest.Mock).mock.calls).toEqual([
 			[
-				'//performance-events.code.dev-guardianapis.com/commercial-metrics',
-				JSON.stringify({
-					browser_id: 'browser id',
-					page_view_id: 'page view id',
-					received_timestamp: '2021-01-01T12:00:00.000Z',
-					received_date: '2021-01-01',
-					platform: 'NEXT_GEN',
-					metrics: [],
-					properties: [{ name: 'isDev', value: 'localhost' }],
-				}),
+				'//performance-events.guardianapis.com/commercial-metrics',
```

Comment:
We could extract this in a constant, like `PROD_ENDPOINT`.

Reply:
Non-blocking: might be worth doing the same here? https://github.com/guardian/commercial-core/pull/267/files#diff-c076222fcae3be8e833dc2a29ce1870cce642fe2aab8c5504875182b46de95c6R34

Reply:
Yes, and then exporting that `const` to reuse it in the test!